galley/pkg/config/analysis/analyzers/util/constants.go

```diff
@@ -19,11 +19,14 @@ import (
 )
 
 const (
-	DefaultKubernetesDomain = "svc.cluster.local"
-	MeshGateway             = "mesh"
-	ExportToNamespaceLocal  = "."
-	ExportToAllNamespaces   = "*"
-	Wildcard                = "*"
+	DefaultKubernetesDomain    = "svc.cluster.local"
+	MeshGateway                = "mesh"
+	ExportToNamespaceLocal     = "."
+	ExportToAllNamespaces      = "*"
+	Wildcard                   = "*"
+	MultiClusterSecretLabelKey = "istio/multiCluster"
```

Comment:
can we get this from the source instead of duplicating it?

Reply:
Done.